app/controllers/hq_controller.rb

```diff
@@ -1,7 +1,7 @@
 class HqController < ApplicationController
   NAVIGATION_BUTTONS= [
     { text: 'OSRA', href: 'root_path', path_regex: /^\/$/, glyph: 'glyphicon-home' },
-    { text: 'Dashboard', href: '"/404.html"', path_regex: /^\/hq(\/)?$/, glyph: 'glyphicon-dashboard' },
+    { text: 'Dashboard', href: '"dashboard"', path_regex: /^\/hq\/dashboard/, glyph: 'glyphicon-dashboard' },
```

Comment:
You can change `"dashboard"` with  `hq_dashboard_path` since you created the route for it. If you are not sure about the route please take a look at `rake routes`.